src/Pim/Component/Catalog/Builder/ProductTemplateBuilder.php

```diff
@@ -64,55 +42,42 @@ public function createProductTemplate()
     /**
      * {@inheritdoc}
      */
-    public function addAttributes(ProductTemplateInterface $template, array $attributes, $locale)
+    public function addAttributes(ProductTemplateInterface $template, array $attributes)
```

Comment:
This method is currently not used anywhere in the PIM since the pefization of the variant groups.
We only keep it if it is used in projects.